leshan-core/src/main/java/org/eclipse/leshan/core/model/ObjectLoader.java

```diff
@@ -39,6 +39,11 @@
 
     private static final Gson GSON;
 
+    private static String[] ddfpaths = new String[] { "LWM2M_Security-v1_0.xml", "LWM2M_Server-v1_0.xml",
+                            "LWM2M_Access_Control-v1_0.xml", "LWM2M_Device-v1_0.xml",
+                            "LWM2M_Connectivity_Monitoring-v1_0.xml", "LWM2M_Firmware_Update-v1_0.xml",
+                            "LWM2M_Location-v1_0.xml", "LWM2M_Connectivity_Statistics-v1_0.xml" };
```

Comment:
As you said, this is additional specs. So not embedded by default, but you can easily add it manually. (like we do in demos app).